keyboards/boardsource/equals/config.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 Cole Smith (@boardsource)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 200U
+
+#define RGB_MATRIX_LED_COUNT 70
+#define RGB_DI_PIN GP21
+
+#define AUDIO_PIN GP29
+#define AUDIO_PWM_DRIVER PWMD6
+#define AUDIO_PWM_CHANNEL RP2040_PWM_CHANNEL_B
+
+#define AUDIO_INIT_DELAY 1000
```

Comment:
This isn't a value, it's either enabled or not.
```suggestion
#define AUDIO_INIT_DELAY
```